libs/sdk-ui-kit/src/Icon/icons/LegendMenu.tsx

```diff
@@ -16,7 +16,11 @@ export const LegendMenu: React.FC<IIconProps> = ({ color, className, width, heig
             xmlns="http://www.w3.org/2000/svg"
         >
             <g stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
-                <g transform="translate(-5.500000, -6.000000)" fill={color ?? "#6D7680"}>
+                <g
+                    id="LegendIcoContent"
```

Comment:
Should it be `id`? If I have multiple charts, id wont be unique?

Reply:
This was there even before but it was deleted for some reason. It was used for properly selecting the inner part and changing its color by css. Was not working without it. Proper css specificity should make it safe.